webpack/scenes/RedHatRepositories/index.scss

```diff
@@ -0,0 +1,9 @@
+$pf-black-200: #ededed;
+
+.background-container-gray {
```

Comment:
I'm definitely open to suggestion on the class names :)

Reply:
I'm not usually for describing the properties of the class in the class name but I don't know what else to call this in this case.